plugins/talk-plugin-like/client/LikeButton.js

```diff
@@ -35,7 +35,7 @@ class LikeButton extends React.Component {
     return (
       <div className={cn(styles.container, `${plugin}-container`)}>
         <button
-          className={cn(styles.button, {[styles.liked]: alreadyReacted}, `${plugin}-button`)}
+          className={cn(styles.button, {[`${styles.liked} talk-plugin-like-liked`]: alreadyReacted}, `${plugin}-button`)}
```

Comment:
Just a tiny thing here:

There's no need to interpolate here. `cn` is just for that, you can just add ` talk-plugin-like-liked` at the end with ','